### PR TITLE
Add model datamart and related initdb

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.PHONY: check
+check: flake8
+
+.PHONY: flake8
+flake8: .build/dev-requirements.timestamp .build/flake8.timestamp
+
+.PHONY: test
+test: .build/dev-requirements.timestamp
+	.build/venv/bin/nosetests
```
Comment (pgiraud) [inline Makefile]: Let's remove the `test` target until we really need it.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+.build/dev-requirements.timestamp: .build/venv dev-requirements.txt
+	mkdir -p $(dir $@)
+	.build/venv/bin/pip install -r dev-requirements.txt > /dev/null 2>&1
+	touch $@
+
+.build/requirements.timestamp: .build/venv setup.py requirements.txt
+	mkdir -p $(dir $@)
+	.build/venv/bin/pip install -r requirements.txt
+	touch $@
```
Comment (pgiraud) [inline Makefile]: This target is not used. I'd suggest we remove it.

File: dev-requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#
+# This file defines the requirements.txt for development.
+#
+nose
```
Comment (pgiraud) [inline dev-requirements.txt]: We don't have any test. So nose is not required (for now).

**Comment by pgiraud** [comment]
I added some minor comments. Otherwise it looks good to me.

**Comment by fvanderbiest** [comment]
Yes LGTM too. Thanks.

**Comment by arnaud-morvan** [comment]
Done

**Comment by pgiraud** [comment]
Please merge.

**Pull request merged**